index.html

```diff
@@ -288,7 +288,12 @@ <h1>Fun and Tools</h1><hr>
 ============================================= ------>
 <div class="footer-container">
     <footer>
-        <p>All ©️copyrights reserved ThePokedex 2023...</p> 
+        <!-- Linking the site to the logo -->
+        <a href="https://the-pokedex-nu.vercel.app/">
+            <img src="https://user-images.githubusercontent.com/62764275/282056861-444edccd-a670-421a-a1f0-45caed15582d.svg" alt="GO Hub Logo" id="footer-logo">
```

Comment:
you can upload footer logo image file and upload it in the images folder by downloading it in the issue #45